src/block-manager.js

```diff
@@ -290,6 +297,31 @@ Object.assign(BlockManager.prototype, require('./function-bind'), require('./med
     }
   },
 
+  paste: function(blocks) {
```

Comment:
Can this paste over a selection of multiple blocks, or is selecting multiple blocks not something that can be done currently? Was thinking it should be `utils.getBlocksBySelection()` to get all blocks that are selected.

Reply:
The is used in as part of pasting when a selection has already been removed. as you'll see below.

```
let data = JSON.parse(ev.clipboardData.getData(TYPE));
if (this.selecting) this.delete();
this.mediator.trigger("block:paste", data);
```